code/game/objects/structures/under_wardrobe.dm

```diff
@@ -1,25 +1,61 @@
+#define WARDROBE_BLIND_MESSAGE(fool) "\The [src] flashes a light at \the [fool] as it states a message."
+
 /obj/structure/undies_wardrobe
 	name = "underwear wardrobe"
 	desc = "Holds item of clothing you shouldn't be showing off in the hallways."
 	icon = 'icons/obj/closet.dmi'
 	icon_state = "cabinet_closed"
```

Comment:
It's invisible in game.

Reply:
You mean the wardrobe ? its missing icon ?

Reply:
Yep.

Reply:
Probably that's related to TG closets sprites.

Reply:
should be fixed